tutorials/source-modeling/plot_source_space_SNR.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-mne-sourcespace-snr:
+
+===============================
+Computing source space SNR
```

Comment:
I think this should be moved to examples folder. It's not a full-fledged tutorial ...